mmcv/ops/point_sample.py

```diff
@@ -247,7 +255,10 @@ def rel_roi_point_to_rel_img_point(rois,
     return rel_img_point
 
 
-def point_sample(input, points, align_corners=False, **kwargs):
+def point_sample(input: Tensor,
+                 points: Tensor,
+                 align_corners: bool = False,
+                 **kwargs: str) -> Tensor:
```

Comment:
we don't need to add type hints for `kwargs` or `args`

Reply:
Okay, I'll fix it tomorrow. Thx